docs/guide/README.md

```diff
@@ -1,94 +1,94 @@
-# Introduction to Vue Storefront
+# General Information
 
-Vue Storefront is a rather complex solution with a lot of possibilities. Learning all of them can take some time. In this introduction, we will learn all of its crucial concepts in a few minutes, which is enough to start playing with Vue Storefront right after reading.
+## Introduction to Vue Storefront
 
+Vue Storefront is rather a complex solution with a lot of possibilities. Learning all of them can take some time. In this introduction, we will learn all its crucial concepts in a few minutes which are enough to start playing with Vue Storefront right after reading it.
 
 ## What is Vue Storefront
-Vue Storefront is a headless and backend-agnostic eCommerce Progressive Web App (PWA) written in Vue.js. The fact that it's using headless architecture allows Vue Storefront to connect with any eCommerce platform so it can be a frontend PWA for Magento, Shopify, BigCommerce, WooCommerce, etc.
 
-Key features of Vue Storefront:
-- Platform-agnostic
-- Focused on performance
-- Mobile-first approach
-- Cutting-edge tech
-- No limitations in theming and customization
-- Open Source with MIT license
-- High-quality developer experience
-- Out-of-the-box Server Side Rendering (for SEO)
-- Offline mode
+Vue Storefront is a headless and backend-agnostic eCommerce Progressive Web App written in Vue.js. The fact that it's using headless architecture allows Vue Storefront to be connected with any eCommerce platform so it can be a frontend PWA for Magento, Shopify, BigCommerce, WooCommerce etc.
 
+Key features of Vue Storefront are:
 
-## How it connect with backend platforms?
-Vue Storefront manages to be platform-agnostic thanks to the [vue-storefront-api](https://github.com/DivanteLtd/vue-storefront-api) and dedicated API connectors for eCommerce backend platforms. The data format in vue-storefront-api is always the same for any platform, which means no matter what eCommerce backend you use, your frontend remains without changes.
+- being platform-agnostic
+- focus on performance
+- mobile-first approach
+- cutting-edge tech
+- no limitations in theming and customization
+- being Open Source with MIT license
+- a lot of focus on developer experience
+- out of the box Server-Side Rendering (for SEO)
```

Comment:
"out-of-the-box" (hyphenated) was correct, please change back

Reply:
Got it